src/components/Onboarding/Steps/StepsPanel.js

```diff
@@ -52,36 +64,23 @@ function StepsPanel() {
           height: ${25 * GU}px;
         `}
       >
-        <CircleGraph
-          value={displayedSteps[step][1] / (displayedStepsCount - 1)}
-          size={25 * GU}
-        />
-        <div
-          css={`
-            position: absolute;
-            top: 130px;
-            font-size: 20px;
-            color: #8e97b5;
-            opacity: 0.7;
-          `}
-        >
-          {`${displayedSteps[step][1] + 1}/${displayedStepsCount}`}
-        </div>
+        <CircleGraph value={step / (Screens.length - 1)} size={25 * GU} />
```

Comment:
```suggestion         	
        <CircleGraph value={step / Screens.length} size={25 * GU} />

Reply:
After the new changes, I notice it was displaying wrong. Going back to your first implementation worked fine.

Reply:
I'll go ahead and commit this change and merge the PR 🙌